tests/tests/lightspeed-install.cy.ts

```diff
@@ -52,43 +52,46 @@ describe('Lightspeed related features', () => {
         oauthorigin,
       );
     });
-  });
-
-  after(() => {
-    // Delete entire namespace to delete operator and ensure everything else is cleaned up
-    cy.adminCLI(`oc delete namespace ${OLS.namespace}`, { failOnNonZeroExit: false });
-
-    // Delete config
-    cy.exec(
-      `oc delete ${OLS.config.kind} ${OLS.config.name} -n ${OLS.namespace} --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
-    );
-
-    cy.adminCLI(
-      `oc adm policy remove-cluster-role-from-user cluster-admin ${Cypress.env('LOGIN_USERNAME')}`,
-    );
-  });
-
-  it(
-    '(OLS-427,jfula,Lightspeed) Deploy OpenShift Lightspeed operator via web console',
-    { tags: ['e2e', 'admin', '@smoke'] },
-    () => {
-      if (Cypress.env('BUNDLE_IMAGE')) {
-        cy.exec(
-          `oc create namespace ${OLS.namespace} --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
-        );
-        cy.exec(
-          `operator-sdk run bundle --timeout=5m --namespace ${OLS.namespace} ${Cypress.env('BUNDLE_IMAGE')} --kubeconfig ${Cypress.env('KUBECONFIG_PATH')} --verbose `,
-          { timeout: 6 * 60 * 1000 },
-        );
-      } else {
-        operatorHubPage.installOperator(OLS.packageName, 'redhat-operators');
-        cy.get('.co-clusterserviceversion-install__heading', { timeout: 5 * 60 * 1000 }).should(
-          'include.text',
-          'ready for use',
-        );
-      }
-
-      const config = `apiVersion: ols.openshift.io/v1alpha1
+    if (Cypress.env('BUNDLE_IMAGE')) {
+      cy.exec(
+        `oc create namespace ${OLS.namespace} --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
+      );
+      cy.exec(
+        `oc label namespaces ${OLS.namespace} openshift.io/cluster-monitoring=true --overwrite=true --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
+      );
+      cy.exec(
+        `operator-sdk run bundle --timeout=20m --namespace ${OLS.namespace} ${Cypress.env('BUNDLE_IMAGE')} --kubeconfig ${Cypress.env('KUBECONFIG_PATH')} --verbose `,
+        { timeout: 6 * 60 * 1000 },
+      );
+    } else {
+      operatorHubPage.installOperator(OLS.packageName, 'redhat-operators');
+      cy.get('.co-clusterserviceversion-install__heading', { timeout: 5 * 60 * 1000 }).should(
+        'include.text',
+        'ready for use',
+      );
+    }
+    if (Cypress.env('CONSOLE_IMAGE')) {
+      // If console image exists, replace it in csv
+      cy.exec(
+        `oc scale --replicas=0 deployment/lightspeed-operator-controller-manager --namespace=${OLS.namespace} --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
+      );
+      cy.exec(
+        `oc patch csv lightspeed-operator.v0.2.1 --namespace=openshift-lightspeed --type='json' -p='[{"op": "replace", "path": "/spec/relatedImages/1/image", "value":"${Cypress.env('CONSOLE_IMAGE')}"}]' --kubeconfig ${Cypress.env('KUBECONFIG_PATH')}`,
```

Comment:
Should we put the `v0.2.1` in an environment variable?

Reply:
I was discussing this with Xavier yesterday. We can maybe do even better. We can check the name of the existing csv prefixed with "lightspeed-operator" and use that name for the patch command